src/cjs/server.js

```diff
@@ -21,9 +21,42 @@ app.use(express.static("game"));
 
 const matchByPlayerToken = {};
 
+app.get("/api/match", async (req, res) => {
+  try {
+    const { playerToken } = req.query;
+    if (typeof playerToken !== "string") {
+      res.status(400).json({
+        message: "playerToken is missing",
+      });
+      return;
+    }
```

Comment:
Looks like this could be moved to a middleware maybe?

Reply:
Good idea, but I am not sure if all routes requires a `playerToken` in the future.